### PR TITLE
Added spanish translation of autoclass_tutorial.

### PR DESCRIPTION
**Pull request by duedme**
# Translation of autoclass_tutorial.mdx into spanish

I made the translation of autoclass_turorial.mdx into Spanish (fixes #15947). The document is located in the docs/source/es folder.

This PR also includes the translation of _toctree.yml to include autoclass_tutorial.

FYI @omarespejel 

## Before submitting
- [X] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [X] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?

### CODE REVIEW

**Review by omarespejel — APPROVED** [review]

File: docs/source/es/_toctree.yml [hunk 1]
```diff
@@ -7,6 +7,8 @@
 - sections:
   - local: pipeline_tutorial
     title: Pipelines para inferencia
+  - local: autoclass_tutorial
+    title: Carga las instancias preentrenadas con un AutoClass
```
Comment (omarespejel) [inline docs/source/es/_toctree.yml:11]:
```suggestion
    title: Carga instancias preentrenadas con un AutoClass
```

**Review by omarespejel — APPROVED** [review]

**Comment by omarespejel** [comment]
LGTM! Just that little change in the `_toctree.yml`, @Duedme. Thanks!

Is it ok if I merge when the change is made @sgugger?

**Comment by omarespejel** [comment]
LGTM! Just that little change in the `_toctree.yml`, @Duedme. Thanks!

Is it ok if I merge when the change is made @sgugger?

**Review by omarespejel — APPROVED** [review]

**Review by sgugger — APPROVED** [review]
Sure thing! Thanks for the PR @Duedme !

**Pull request merged**